utils/src/main/kotlin/Ci.kt

```diff
@@ -24,5 +24,8 @@ package org.ossreviewtoolkit.utils
  */
 object Ci {
     val isAppVeyor = listOf("APPVEYOR", "CI").all { Os.env[it]?.toBoolean() == true }
+    val isAzure = Os.env["AGENT_OS"] != null
+    val isAzureLinux = Os.env["AGENT_OS"] == "Linux"
+    val isAzureWindows = Os.env["AGENT_OS"] == "Windows_NT"
```

Comment:
If the above is changed, these should include `isAzure &&`.

Reply:
Done.